pkg/reconciler/pubsub/topic/topic.go

```diff
@@ -444,3 +250,14 @@ func (r *Reconciler) UpdateFromTracingConfigMap(cfg *corev1.ConfigMap) {
 	r.Logger.Debugw("Updated Tracing config", zap.Any("tracingCfg", r.tracingConfig))
 	// TODO: requeue all Topics. See https://github.com/google/knative-gcp/issues/457.
 }
+
+func (r *Reconciler) FinalizeKind(ctx context.Context, topic *v1alpha1.Topic) reconciler.Event {
+	if topic.Spec.PropagationPolicy == v1alpha1.TopicPolicyCreateDelete {
+		logging.FromContext(ctx).Desugar().Debug("Deleting Pub/Sub topic")
+		if err := r.deleteTopic(ctx, topic); err != nil {
+			return reconciler.NewEvent(corev1.EventTypeWarning, deleteTopicFailed, "Failed to delete Pub/Sub topic: %s", err.Error())
+		}
+		topic.Status.TopicID = ""
```

Comment:
I think this can be removed, because it will try to update the status otherwise, and it's not needed when it it will be deleted...

Reply:
@grac3gao I'll get this one in, can you address this cosmetic comment in your PR once you rebase?